backend/instancer/api/account.py

```diff
@@ -336,6 +336,30 @@ def login() -> ResponseReturnValue:
     }
 
 
+@blueprint.route("/logout", methods=["POST"])
+def logout() -> ResponseReturnValue:
+    """Logout the current user
+
+    Requires body param login_token"""
+    try:
+        token = request.form["login_token"]
+    except KeyError:
+        return {"status": "missing_login_token", "msg": "Missing login token"}, 401
+    try:
+        account = LoginToken.decode(token)
```

Comment:
`LoginToken` class is for login tokens and contain information that is encrypted and digitally signed, and once used to login, is not saved on the client in react - session (or account) tokens is what's then used for authentication, and are just random strings of data that can be correlated with an account in redis. You'll want to instead just pass the session token directly into authentication.del_session() to invalidate the session